npm/test/clientfileprocessing.test.ts

```diff
@@ -255,12 +254,84 @@ test("file successfully uploaded to s3", async () => {
   const s3UploadMock = new S3UploadMock()
   const fileProcessing = new ClientFileProcessing(metadataUpload, s3UploadMock)
   await expect(
-    fileProcessing.processClientFiles("1", [], jest.fn(), "")
+    fileProcessing.processClientFiles("1", [], "")
   ).resolves.not.toThrow()
 
   expect(s3UploadMock.uploadToS3).toHaveBeenCalledTimes(1)
 })
 
+test("s3ProgressBarCallback function updates the progress bar with the percentage processed", () => {
```

Comment:
Name the function in the test description as the actual function name: "s3ProgressCallback"